lib/gruf/instrumentation/request_logging/hook.rb

```diff
@@ -139,6 +141,29 @@ def sanitize(params = {})
         def options
           super().fetch(:request_logging, {})
         end
+
+        def redact!(parts, i, params, redacted_string)
```

Comment:
Wonder if it makes sense to move this implementation to its own class to simplify testing.